ibis/backends/pandas/executor.py

```diff
@@ -593,11 +593,18 @@ def visit(cls, op: ops.Sort, parent, keys):
         # 2. sort the dataframe using those columns
         # 3. drop the sort key columns
         ascending = [key.ascending for key in op.keys]
+        na_pos_dict = {True: "first", False: "last"}
```

Comment:
I'm not sure this is the best way to pass `na_position`. Same hack than in the dask executor.